include/layer_shell_v1.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2018-2019 Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: William Wold <william.wold@canonical.com>
+ */
+
+#ifndef WLCS_LAYER_SHELL_V1_H
+#define WLCS_LAYER_SHELL_V1_H
+
+#include "in_process_server.h"
+
+// Because _someone_ *cough*ddevault*cough* thought it would be a great idea to name an argument "namespace"
```

Comment:
Just maybe it is the generator at fault, why should a protocol definition care what the keywords are in all possible implementation languages?